lib/chat/chat_messenger.dart

```diff
@@ -157,49 +158,63 @@ class ChatMessengerState extends State<ChatMessenger> {
     return Container(
       decoration: BoxDecoration(
         color: Colors.white,
-        border: Border(top: BorderSide(width: 1.0, color: Colors.grey[100]))
       ),
-      constraints: BoxConstraints(minHeight: 70.0, maxHeight: 200.0),
-      padding: EdgeInsets.only(bottom: 15.0),
+      constraints: BoxConstraints(minHeight: 50.0, maxHeight: 200.0),
+      padding: EdgeInsets.all(15.0),
       child: Row(
         mainAxisAlignment: MainAxisAlignment.spaceEvenly,
         crossAxisAlignment: CrossAxisAlignment.end,
         children: <Widget>[
-          IconButton(
-            icon: Icon(Icons.photo),
-            iconSize: 25.0,
-            color: Theme.of(context).primaryColor,
-            onPressed: () => pickImage(ImageSource.gallery),
-          ),
-          IconButton(
-            icon: Icon(Icons.camera),
-            iconSize: 25.0,
-            color: Theme.of(context).primaryColor,
-            onPressed: () => pickImage(ImageSource.camera),
-          ),
           Expanded(
-            child: TextField(
-              textCapitalization: TextCapitalization.sentences,
-              decoration: InputDecoration.collapsed(
-                hintText: 'Send a message...'
-              ),
-              style: GoogleFonts.muli(
-                textStyle: TextStyle(
-                  color: Colors.black, 
-                  letterSpacing: .5, 
-                  fontSize: 14.0, 
-                )
+            child: Container(
+              decoration: BoxDecoration(
+                border: Border.all(width: 1.0, color: Colors.grey[300]),
               ),
-              maxLines: null, // this allows for multi-line input
-              textInputAction: TextInputAction.send,
-              controller: textInputController,
-              onSubmitted: (value) => submitMessage(textMessage, value),
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.end,
+                children: <Widget>[
+                  IconButton(
+                    icon: Icon(Icons.photo),
+                    iconSize: 24.0,
+                    padding: EdgeInsets.all(0.0),
+                    color: Theme.of(context).accentColor,
+                    onPressed: () => pickImage(ImageSource.gallery),
+                  ),
+                  IconButton(
+                    icon: Icon(Icons.camera_alt),
+                    iconSize: 24.0,
+                    padding: EdgeInsets.all(0.0),
+                    color: Theme.of(context).accentColor,
+                    onPressed: () => pickImage(ImageSource.camera),
+                  ),
+                  Expanded(
+                    child: TextField(
+                      textCapitalization: TextCapitalization.sentences,
+                      decoration: InputDecoration.collapsed(
+                        hintText: 'Send a message...',
+                      ),
+                      style: GoogleFonts.muli(
```

Comment:
Can use the `FormatText` widget here.

Reply:
don't think I can, this is a `TextField` whereas `FormatText` is just a `Text`

Reply:
I think aside from `FormatText`, we should also make a general `FormatTextStyle` widget since I've also been needing it lately for my form fields  👀 So ya, you can leave this be for now! @nayancprakash